ckan/.snyk

```diff
@@ -37,34 +37,6 @@ ignore:
           https://github.com/GSA/data.gov/issues/4303
         expires: 2024-03-31T16:20:58.017Z
         created: 2023-05-08T16:20:58.023Z
-  SNYK-PYTHON-WERKZEUG-6041510:
```

Comment:
I don't see a werkzeug version bump. 

Did this resolve the other werkzeug exception on L19 as well?

What about the Flask one on L33?

Reply:
did not evaluate what is fixed. Just evaluated what is not needed. I removed this exception, snyk did not complain.